src/IotCentral/IotCentral/Az.IotCentral.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.9.0'
+ModuleVersion = '1.0.0'
```

Comment:
- Module version is calculated, so please revert this change
- `1.0.0` means this powershell module is GA. GA is a big deal so if this is not a mistake, please refer to [Process of GA](https://github.com/Azure/azure-powershell-pr/wiki/Process-of-GA-module)

Reply:
Done